lib/src/kvs/tests/nq.rs

```diff
@@ -20,7 +18,7 @@ async fn archive_lv_for_node_archives() {
 	let key = crate::key::node::lq::new(node_id, lv_id.0, namespace, database);
 	tx.putc(key, table, None).await.unwrap();
 
-	let (_, mut stm) = live(format!("LIVE SELECT * FROM {}", table).as_str()).unwrap();
+	let mut stm = LiveStatement::from_source_parts(Fields::all(), Table(table.into()), None, None);
```

Comment:
Why did this change? The original was used delierately as a different approach from LiveStatement (to have variety in usages of statements). Happy to keep the modification. Just surprised and wondered what the motivation was.

Reply:
The `live` function is now private inside `syn` so I had to changes this. 

I was thinking about the future where the module will be in it's own crate, we would have to then make the `live` function public just for this test which I disliked so instead I kept it private.

Reply:
I would also have to support this `live` function in the new parser when it gets merged.

Reply:
There where some other test which also called otherwise private parts of the parser. I just tried to reorganize most of these calls to use more general function so we can define a more limited public interface.

Reply:
Yeah that's all fine, agree with the approach now :)